frontend/tests/acceptance/step_definitions/dashboardContext.js

```diff
@@ -0,0 +1,114 @@
+const { I } = inject();
+const dashboard = require('../pages/dashboardPage');
+const { addJobApplication, deleteJobApplication } = require('../helpers/api/job_application');
+const USER_API = require('../helpers/api/user');
+const updateJob = require('../pages/updateJobApplicationPage');
+
+let Token, firstJob, secondJob;
```

Comment:
IMO, it is better variable name should start from lower case letter to follow the convention on this project. Rename the variable `Token` .